index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Consider having a readme file

Reply:
I already had the readme file in my repo, no? (unless I am missing something).